data/actionsmap/yunohost.yml

```diff
@@ -1477,6 +1477,39 @@ tools:
 
 
 #############################
+#         Migrations        #
+#############################
+migrations:
+    category_help: Manage migrations
+    actions:
+
+        ### migrations_list()
+        list:
+            action_help: List migrations
+            api: GET /migrations
+
+        ### migrations_migrate()
+        migrate:
```

Comment:
Can we rename this 'apply' ? :D 

`yunohost migrations apply` seems more beautiful than `yunohost migrations migrate`